src/modules/AutoinstDrive.ycp

```diff
@@ -330,20 +330,28 @@
   */
   global define DriveT
     parseDrive( map drive ){
+
     DriveT newDrive = new( "auto", drive["type"]:`CT_DISK );
     newDrive = set( newDrive, "device", drive["device"]:"auto" );
     newDrive = set( newDrive, "initialize", drive["initialize"]:true );
     newDrive = set( newDrive, "use", string2symbol( drive["use"]:"all" ));
     newDrive = set( newDrive, "pesize", drive["pesize"]:"" );
-    foreach( map part, drive["partitions"]:[], {
-      PartitionT newPart = AutoinstPartition::parsePartition( part );
-      if( AutoinstPartition::isPartition( newPart ) ){
-	newDrive = addPartition( newDrive, newPart );
-      }
-      else{
-	y2error("Couldn't construct PartitionT from '%1'", part );
-      }
-    });
+    if( drive["type"]:`x == `CT_NFS ){
+      newDrive["initialize"] = false; // Do not initialize
+      # If it is a root nfs partition we do not need any additional
+      # partition conversation. (bnc#986124)
+      newDrive["partitions"] = drive["partitions"]:[];
```

Comment:
The above code uses a `set` function but these 2 lines use direct assignment. Why the inconsistency?